zyte_common_items/base.py

```diff
@@ -118,3 +121,52 @@ def _apply_field_types_to_sub_fields(cls, item: Optional[Dict]):
             item.update({key: cls.from_list(item.get(key, [])) for key, cls in (from_list or {}).items()})
 
         return item
+
+
+@export
+@attrs.define(slots=True)
+class AdditionalProperty:
+    name: str
+    value: str
+
+
+@export
+@attrs.define(slots=True, kw_only=True)
+class AggregateRating:
+    best_rating: Optional[float] = None
+    rating_value: Optional[float] = None
```

Comment:
In general, for `floats`, do we use `float`, or `Decimal`? `Decimal` would preserve the API output level of detail, although for most fields I am not sure this is important enough.

Reply:
I think we're using floats, unless we really want to preserve the precision, like in price. For aggregate rating value it doesn't matter, we should use float here.